2017/Cambridge-OMERO-API/index.html

```diff
@@ -0,0 +1,268 @@
+                </section>
+
+
+                <section>
+                    <h2>OMERO model objects</h2>
+                    <img src="images/developer-model-pdi.png">
+                    <br>
+                    See <a href="http://docs.openmicroscopy.org/omero/5.4.0/developers/Model/EveryObject.html">
+                        All Objects
+                    </a>
+                </section>
+
+
+
+                <section>
+                    <h2>Remote Objects & Proxy Services</h2>
+                    <img src="images/omero-proxy-objects.png"><br>
+                    <a href="http://docs.openmicroscopy.org/latest/omero/developers/index.html">
+                        Developer Documentation
+                    </a>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python setup</h2>
+                    <ul>
+                        <li>Download
+                            <a href="https://downloads.openmicroscopy.org/latest/omero5/py.zip">
+                                OMERO.py libs
+                            </a>, add to PYTHONPATH
+                        </li>
+                        <li>
+                            <code class="python">$ pip install zeroc-ice&gt;3.5,&lt;3.7</code>
+                        </li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python code</h2>
+                    <pre><code class="python">import omero
+
+client = omero.client("localhost")
+# login to create a session (Service Factory)
+session = client.createSession("username", "password")
+# get a Stateless Service
+queryService = session.getQueryService()
+# get a local omero.model.DatasetI
+dataset = queryService.get("Dataset", 1)
+print dataset               # see what's loaded
+print dataset.name.val      # rtypes same as getName().getValue()
+print dataset.copyImageLinks()      # omero.UnloadedEntityException!
+
+client.closeSession()           # Free server resourses
+</code></pre>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Java Setup</h2>
+                    <ul>
+                        <li>Download
+                            <a href="https://downloads.openmicroscopy.org/latest/omero/artifacts">
+                                OMERO.java libs
+                            </a>, add to Java classpath
+                        </li>
+                        <li>Or use Maven project: see our <a href="https://docs.openmicroscopy.org/latest/omero/developers/Java.html">
+                                Java
+                            </a> docs page.
+                        </li>
+                    </ul>
+
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Java Code</h2>
+                    <pre><code class="java">import omero.api.IQueryPrx;
```

Comment:
same comment about ``keepAlive``

Reply:
I'm not sure I want to mention keepAlive at this stage. It's not essential for getting started and I'd like to keep things as simple as possible.

Reply:
It will need to be mentioned at some point, during the hands on session?